people/diannasoreil.toml

```diff
@@ -1,5 +0,0 @@
-name = "Liv"
```

Comment:
FYI CI is failing because you deleted the toml file instead of moving it -- perhaps you forgot to commit the renamed file?

Reply:
oh, maybe, I did use `git commit -a`, let me amend...